<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonWrite">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>87</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonrRead">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>87</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sourcelineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="destlineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copypushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
